core/src/i18n/en.yaml

```diff
@@ -164,6 +170,29 @@ daemon-sets:
     title: Tolerations
     toleration-seconds: Seconds
 deployments:
+  create-modal:
+    advanced:
+      cpu-limits: CPU limits
+      cpu-requests: CPU requests
+      expose-service: Expose separate Service
+      memory-limits: Memory limits
+      memory-requests: Memory requests
+      port: Port
+      port-placeholder: Enter port at which expose the Service
```

Comment:
Does the Service expose the port, or does the port expose the Service?
Depending on that, I'd suggest either

`port-placeholder: Enter the port that exposes the Service`

or

`port-placeholder: Enter the port exposed by the Service`